authors/apps/articles/models.py

```diff
@@ -0,0 +1,49 @@
+from django.db import models
+from django.utils.text import slugify
+
+from authors.apps.authentication.models import User
+from cloudinary.models import CloudinaryField
+
+
+class Article(models.Model):
+    """
+    The model for creating the articles is defined here
+    """
+    slug = models.SlugField(
+        db_index=True, max_length=1000, unique=True, blank=True)
+    title = models.CharField(max_length=1000, blank=False)
+    description = models.CharField(max_length=2000, blank=False)
+    body = models.TextField(blank=False)
+    image_url = CloudinaryField(blank=True, null=True)
```

Comment:
Ensure you save both the image and the absolute image URL after the image is uploaded to cloudinary.